examples/benchmarks/bubblebench/baseline/src/state.ts

```diff
@@ -2,6 +2,7 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
```

Comment:
If disabling a lint, its best practice to document why this the relevant code (in this case this entire file) needs to do something our linting policy says is bad, and why its actually ok in this specific case.